hcc_config/hcc_config.cpp

```diff
@@ -166,6 +166,8 @@ void ldflags(void) {
     std::cout << " -L" XSTR(CODEXL_ACTIVITY_LOGGER_LIBRARY);
     std::cout << " -lCXLActivityLogger";
 #endif
+
+    std::cout << " -lhc_profile";
```

Comment:
I'd suggest adding a cmake flag to control the enablement of hc_profile and let it be OFF by default for now.  Once the profiler package becomes available in a future release of rocm, we could consider turning it ON by default and make that a dependency for hcc.